src/main/webapp/new_post.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="UTF-8">
+    <title>New Post</title>
```

Comment:
The same thing applies here. A better title will be sign up page or sign in page.